From 3af6e5e03288d162c94165f77ed3878b58790016 Mon Sep 17 00:00:00 2001
From: rkannapp <r.kannappan@intel.com>
Date: Thu, 3 Jul 2014 09:48:14 +0530
Subject: [PATCH] EM/Battery: Reduce uevent during resume

BZ: GMINL-6626

Driver's Resume function explictly calls power_supply_changed
which shall result in additional uevent passed to userspace.
This patch calls the battery monitor thread to send the uevent
to the user space.

Change-Id: Ia4d2cae257bd2abd0f81b43a0b25c430be0ea2b2
Signed-off-by: rkannapp <r.kannappan@intel.com>
---
 drivers/power/dc_xpwr_battery.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/power/dc_xpwr_battery.c b/drivers/power/dc_xpwr_battery.c
index 9438ac5..f7b1774 100644
--- a/drivers/power/dc_xpwr_battery.c
+++ b/drivers/power/dc_xpwr_battery.c
@@ -1300,8 +1300,7 @@ static int pmic_fg_resume(struct device *dev)
 
 	dev_dbg(dev, "%s called\n", __func__);
 
-	power_supply_changed(&info->bat);
-	schedule_delayed_work(&info->status_monitor, STATUS_MON_DELAY_JIFFIES);
+	schedule_delayed_work(&info->status_monitor, 0);
 	return 0;
 }
 
-- 
1.9.1

